### PR TITLE
Fix typos in Snowflake policy-recommendation command

### PR DESCRIPTION
**Pull request by jianjuns**
Signed-off-by: Jianjun Shen <shenj@vmware.com>

### CODE REVIEW

File: snowflake/cmd/policyRecommendation.go [hunk 1]
```diff
@@ -204,8 +204,8 @@ var policyRecommendationCmd = &cobra.Command{
 	Use:   "policy-recommendation",
 	Short: "Run the policy recommendation UDF in Snowflake",
 	Long: `This command runs the policy recommendation UDF in Snowflake. 
-You need to bring your own Snowflake account and created the policy
-recommendation UDF using the create-udfs command first.
+You need to bring your own Snowflake account and must have the policy
+recommendation UDF created using the create-udfs command first.
```
Comment (dreamtalen) [inline snowflake/cmd/policyRecommendation.go:208]: Just realized UDF creation has been included in the onboard command. Could you help change it to `must have run the onboard command first`

**Review by dreamtalen — COMMENTED** [review]
Thanks Jianjun.

**Review by dreamtalen — APPROVED** [review]
LGTM

File: snowflake/cmd/policyRecommendation.go (hunk 1 above)
Comment (jianjuns) [reply snowflake/cmd/policyRecommendation.go]: Check. I am not sure if "must have done sth" can also mean what we want here. Let me know if you are sure about that.

Comment (dreamtalen) [reply snowflake/cmd/policyRecommendation.go]: Current version looks good to me, `must have` and `need to` are a bit repetitive.

**Pull request merged**